package.json

```diff
@@ -21,6 +21,9 @@
     "semantic-release": "semantic-release",
     "test": "jest"
   },
+  "dependencies": {
+    "semver-compare": "^1.0.0"
```

Comment:
Lets use the other semver package that is a module and wasn't taken down from the other PR.

Also probably a good time to bump up from node 14 here.